app/controllers/copy_projects_controller.rb

```diff
@@ -86,7 +86,7 @@ def project_copy
   end
 
   def origin
-    params[:coming_from] == 'admin' ? projects_admin_index_path : settings_project_path(@project.id)
+    params[:coming_from] == 'admin' ? projects_path : settings_project_path(@project.id)
```

Comment:
Is that case actually still possible? Where can an admin still copy a project from within the admin area?